src/components/App.js

```diff
@@ -1,68 +0,0 @@
-import React, { useEffect, useState } from "react";
-import LocationDetails from "./LocationDetails";
-import ForecastSummaries from "./ForecastSummaries";
-import ForecastDetails from "./ForecastDetails";
-import SearchForm from "./SearchForm";
-import getForecast from "../requests/getForecast";
-import "../styles/App.css";
-
-function App() {
-  const [selectedDate, setSelectedDate] = useState(0);
-  const [forecasts, setForecasts] = useState([]);
-  const [location, setLocation] = useState({
-    city: "",
-    country: "",
-  });
-  const [searchText, setSearchText] = useState("");
-  const [errorMessage, setErrorMessage] = useState("");
-
-  const selectedForecast = forecasts.find(
-    (forecast) => forecast.date === selectedDate
-  );
-
-  useEffect(() => {
-    getForecast(setSelectedDate, setForecasts, setLocation);
-  }, []);
-
-  const handleCitySearch = () => {
-    getForecast(
-      setSelectedDate,
-      setForecasts,
-      setLocation,
-      searchText,
-      setErrorMessage
-    );
-  };
-
-  const handleForecastSelect = (date) => {
-    setSelectedDate(date);
-  };
-
```

Comment:
You can destructure these like this:
`const { city, country }`
to save yourself typing location. each time